analyzer_test.go

```diff
@@ -105,36 +106,38 @@ var _ = Describe("Analyzer", func() {
 			defer controlPackage.Close()
 			controlPackage.AddFile("md5.go", source)
 			controlPackage.Build()
-			analyzer.Process(buildTags, controlPackage.Path)
+			analyzer.Process(buildTags, controlPackage.Path+"/md5.go")
 			controlIssues, _, _ := analyzer.Report()
 			Expect(controlIssues).Should(HaveLen(sample.Errors))
 
 		})
 
-		It("should report for Golang errors and invalid files", func() {
-			analyzer.LoadRules(rules.Generate().Builders())
-			pkg := testutils.NewTestPackage()
-			defer pkg.Close()
-			pkg.AddFile("foo.go", `
-				package main
-				func main()
-				}`)
-			pkg.Build()
-			err := analyzer.Process(buildTags, pkg.Path)
-			Expect(err).ShouldNot(HaveOccurred())
-			_, _, golangErrors := analyzer.Report()
-			keys := make([]string, len(golangErrors))
-			i := 0
-			for key := range golangErrors {
-				keys[i] = key
-				i++
-			}
-			fileErr := golangErrors[keys[0]]
-			Expect(len(fileErr)).To(Equal(1))
-			Expect(fileErr[0].Line).To(Equal(4))
-			Expect(fileErr[0].Column).To(Equal(5))
-			Expect(fileErr[0].Err).Should(MatchRegexp(`expected declaration, found '}'`))
-		})
+		/*
+			It("should report for Golang errors and invalid files", func() {
+				analyzer.LoadRules(rules.Generate().Builders())
```

Comment:
Again prefer `Skip()` over comment

Reply:
See my comment above in the analyser. I think we should be able to get the errors from Go module loader. 

https://godoc.org/golang.org/x/tools/go/packages#Package

See the error field.